app/views/layouts/application.html.erb

```diff
@@ -10,10 +10,26 @@
 
   <body>
     <nav class = "topnav">
-      <%= link_to "All Merchants", "/merchants"%>
+      <%= link_to "Home", "/"%>
       <%= link_to "All Items", "/items"%>
+      <%= link_to "All Merchants", "/merchants"%>
+      <% if !current_admin? %>
       <%= link_to "Cart: #{cart.total_items}", "/cart" %>
-      <%= link_to "Logout", "/logout", method: :delete %>
+      <% end %>
+      <% if current_user %>
+        <span>Logged in as <%= current_user.name %></span>
```

Comment:
isn't their "username", the user's email? or should we go with name, think both are valid so don't have a preference really

Reply:
Based on the user story, it looks like they wanted the person's name:

_"I also see text that says "Logged in as Mike Dao" (or whatever my name is)"_